script.js

```diff
@@ -0,0 +1,54 @@
+window.document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Do you really need this event listener?

Is there other more optimal way to link script and wait for HTML content load? 🤔

- [ ] Remove `DOMContentLoaded` event listener and implement more optimal solution.